src/server.cpp

```diff
@@ -1211,17 +1234,62 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
                     // mono: copy same mono data in both out stereo audio channels
                     for ( i = 0, k = 0; i < iServerFrameSizeSamples; i++, k += 2 )
                     {
-                        // left/right channel
-                        vecfIntermProcBuf[k]     += vecsData[i];
-                        vecfIntermProcBuf[k + 1] += vecsData[i];
+                        if ( bDelayPan )
+                        {
+                            // pan address shift
+                            // left channel
+                            iLpan = i - iPanDelL;
+                            if ( iLpan < 0 )
+                            {                                            // get from second
+                                iLpan = iLpan + iServerFrameSizeSamples; //
+                                vecfIntermProcBuf[k] += vecsData2[iLpan];
+                            }
+                            else
+                                vecfIntermProcBuf[k] += vecsData[iLpan];
+                            // right channel
+                            iRpan = i - iPanDelR;
+                            if ( iRpan < 0 )
+                            {                                            // get from second
+                                iRpan = iRpan + iServerFrameSizeSamples; //
+                                vecfIntermProcBuf[k + 1] += vecsData2[iRpan];
+                            }
+                            else
+                                vecfIntermProcBuf[k + 1] += vecsData[iRpan];
+                            ;
```

Comment:
What's this for?

Reply:
What do you mean? The spurious semicolon? That can go of course.

Reply:
Yes.

Reply:
How would I have to fix this? Push another commit?

And are the lone else's without braces ok?

Reply:
I would use a force push. Concerning the else I don't think that's the style used overall. But since we're getting improved code, I'd wait for this before working on more here.